tests/5.0/teams_loop/test_target_teams_loop_defaultmap.c

```diff
@@ -0,0 +1,238 @@
+#include <omp.h>
```

Comment:
Add test description here, see format of existing tests.

Reply:
added description